asm/Kptfile

```diff
@@ -212,7 +212,7 @@ openAPI:
       x-k8s-cli:
         substitution:
           name: token-audiences-hub
-          pattern: "istio-ca,ENVIRON_PROJECT_ID.hub.id.goog"
+          pattern: "istio-ca,ENVIRON_PROJECT_ID.svc.id.goog"
```

Comment:
Can we make these changes in https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/pull/694?

Reply:
Done.